devops/sources_backup.rst

```diff
@@ -0,0 +1,122 @@
+.. _conan_backup_sources:
+
+Backup sources
+==============
+
+The *backup sources* feature is intended for storing the downloaded recipe sources in a secondary location,
```

Comment:
Maybe better than secondary, or a clarification of what secondary means, like "like a file server in our own infrastructure"?